samples/java/src/main/java/br/com/example/leadIntegration/controllers/LeadIntegrationController.java

```diff
@@ -0,0 +1,19 @@
+package br.com.example.leadIntegration.controllers;
+
+import org.springframework.web.bind.annotation.*;
+import br.com.example.leadIntegration.models.Lead;
+
+@RestController
+@RequestMapping("/leads")
+public class LeadIntegrationController {
+
+    @GetMapping("/health-check")
```

Comment:
@giovannicaldeira use the `actuator` to create the default springboot health check https://www.baeldung.com/spring-boot-actuators

Reply:
@robsonoduarte added `actuator` to project. Remove `health-check` endpoint.